src/modules/conversation-handler/conversationGpt.ts

```diff
@@ -68,10 +98,10 @@ export async function conversationGpt(
           content: userPrompt!,
           role: "user",
         });
-        ctx.reply(appText.generatingText);
+        await ctx.reply(appText.generatingText);
       }
     }
-    conversation.session.openAi.chatGpt.chatConversation = [...chat];
+    console.log("outside");
```

Comment:
Remove console.log or create more clear log output and use logger, for example "Conversation completed" or something like this

Reply:
Deleted. I was using the console.log as code flags.